docs/spin.toml

```diff
@@ -24,4 +17,188 @@ source = "modules/spin_static_fs.wasm"
+executor = { type = "wagi" }
+
+[[component]]
+id = "redirect-rust-components"
+source = "modules/redirect.wasm"
+environment = { DESTINATION = "https://developer.fermyon.com/spin/rust-components" }
+[component.trigger]
+route = "/rust-components"
+executor = { type = "wagi" }
+
+[[component]]
+id = "redirect-go-components"
+source = "modules/redirect.wasm"
+environment = { DESTINATION = "https://developer.fermyon.com/spin/go-components" }
+[component.trigger]
+route = "/go-components"
+executor = { type = "wagi" }
+
+[[component]]
+id = "redirect-other-languages"
+source = "modules/redirect.wasm"
+environment = { DESTINATION = "https://developer.fermyon.com/spin/other-languages" }
+[component.trigger]
+route = "/other-languages"
+executor = { type = "wagi" }
+
+[[component]]
+id = "redirect-url-shortener"
+source = "modules/redirect.wasm"
+environment = { DESTINATION = "https://developer.fermyon.com/spin/url-shortener" }
+[component.trigger]
+route = "/url-shortener"
+executor = { type = "wagi" }
+
+[[component]]
+id = "redirect-http-trigger"
+source = "modules/redirect.wasm"
+environment = { DESTINATION = "https://developer.fermyon.com/spin/http-trigger" }
+[component.trigger]
+route = "/http-trigger"
+executor = { type = "wagi" }
+
+[[component]]
+id = "redirect-redis-trigger"
+source = "modules/redirect.wasm"
+environment = { DESTINATION = "https://developer.fermyon.com/spin/redis-trigger" }
+[component.trigger]
+route = "/redis-trigger"
+executor = { type = "wagi" }
+
+[[component]]
+id = "redirect-distributing-apps"
+source = "modules/redirect.wasm"
+environment = { DESTINATION = "https://developer.fermyon.com/spin/distributing-apps" }
+[component.trigger]
+route = "/distributing-apps"
+executor = { type = "wagi" }
+
+[[component]]
+id = "redirect-extending-and-embedding"
+source = "modules/redirect.wasm"
+environment = { DESTINATION = "https://developer.fermyon.com/spin/extending-and-embedding" }
+[component.trigger]
+route = "/extending-and-embedding"
+executor = { type = "wagi" }
+
+[[component]]
+id = "redirect-deploying-to-fermyon"
+source = "modules/redirect.wasm"
+environment = { DESTINATION = "https://developer.fermyon.com/spin/deploying-to-fermyon" }
+[component.trigger]
+route = "/deploying-to-fermyon"
+executor = { type = "wagi" }
+
+[[component]]
+id = "redirect-template-authoring"
+source = "modules/redirect.wasm"
+environment = { DESTINATION = "https://developer.fermyon.com/spin/template-authoring" }
+[component.trigger]
+route = "/template-authoring"
+executor = { type = "wagi" }
+
+[[component]]
+id = "redirect-sips-index"
+source = "modules/redirect.wasm"
+environment = { DESTINATION = "https://github.com/fermyon/spin/blob/main/docs/content/sips/index.md" }
+[component.trigger]
+route = "/sips/index"
+executor = { type = "wagi" }
+
+[[component]]
+id = "redirect-sip-001"
+source = "modules/redirect.wasm"
+environment = { DESTINATION = "https://github.com/fermyon/spin/blob/main/docs/content/sips/index.md" }
```

Comment:
Is it intentional to not redirect to the specific SIP? Also for the following redirects.

Reply:
I can go either way on this. I wasn't sure what the best practice was when linking back to GitHub. So I just linked them all to the main SIP index.

Reply:
Fixed